src/runtime/virtcontainers/device/config/config.go

```diff
@@ -263,9 +263,15 @@ type VFIODev struct {
 	// Guest PCI path of device
 	GuestPciPath vcTypes.PciPath
 
+	// APDevices are the Adjunct Processor devices in the mdev. Mutually exclusive with PCI.
```

Comment:
What "in the mdev" means is unclear to me.  It's also unclear why they'd be mutually exclusive with PCI - s390x can have a PCI bus, IIUC, even if it's not particularly common.